doc/source/changes/version_0_29.rst.inc

```diff
@@ -67,6 +67,31 @@ New features
     >>> arr.meta.title = 'array for testing'
 
 
+* allowed sessions to have metadata.
+  Like for arrays, session metadata can be accessed using the syntax ``session.meta.name``.
+  See bellow for use cases:
+
+    >>> # add metadata to a session at creation
+    >>> # Python <= 3.5
+    >>> s = Session([('arr1', ndtest(2)), ('arr2', ndtest(3)], meta=[('title', 'my title'), ('author', 'John Smith')])
+    >>> # Python 3.6+
+    >>> s = Session(arr1=ndtest(2), arr2=ndtest(3), meta=Metadata(title='my title', author='John Smith'))
+    >>> # show/access metadata
+    >>> s.meta
+    title: my title
+    author: John Smith
+
+  Warnings:
+
+  - Excel, CSV and HDF (.h5) file formats support saving and loading session metadata.
```

Comment:
* These are all the common formats, so it is a bit weird to have this in a Warning section, but maybe putting that in a Notes section by itself would be impractical.
* I would emphasize more the difference with arrays by saying something like "Contrary to array metadata, saving and loading session metadata is supported for all current session file formats: Excel, CSV and HDF (.h5).